commondata/CMS_t_tch_13TeV_2016_diff_Yt.yaml

```diff
@@ -1,151 +1,35 @@
 dataset_name: CMS_t_tch_13TeV_2016_diff_Yt
-num_data: 5
-num_sys: 19
+doi: none
+location: Figure 5 (right panel), (normalised -> absolute)
+arxiv: https://inspirehep.net/literature/1456074
+hepdata: none
+units: pb
+description: single-top-quark production in the t-channel differential in the rapidity of the top + anti-top
```

Comment:
is this description correct? It sounds more tt bar ...

Reply:
top + anti-top refers to the sum of single top and single anti-top production

Reply:
okay, thanks for the clarification